cmd/snap-confine/mount-support.c

```diff
@@ -561,6 +562,7 @@ static void sc_bootstrap_mount_namespace(const struct sc_mount_config *config)
 	// pre-pivot filesystem.
 	if (config->distro == SC_DISTRO_CLASSIC) {
 		sc_mount_nvidia_driver(scratch_dir, config->base_snap_name);
+		sc_mount_wsl2_gpu_driver(scratch_dir);
 	}
```

Comment:
and the helper would be used like so:
```suggestion
	if (config->distro == SC_DISTRO_CLASSIC) {
		sc_mount_ensure_extra_lib_dir();
		sc_mount_nvidia_driver(scratch_dir, config->base_snap_name);
		sc_mount_wsl2_gpu_driver(scratch_dir);
	}
```